config/i18n/locales/swahili/translations.json

```diff
@@ -1,67 +1,67 @@
-    "multiple-posts": "A collection of {{ postCount }} posts"
+    "no-posts": "Mkusanyiko wa machapisho",
+    "one-post": "Mkusanyiko wa chapisho 1",
+    "multiple-posts": "Mkusanyiko wa machapisho {{ postCount }} "
   },
   "404": {
-    "page-not-found": "Page not found",
-    "go-to-front-page": "Go to the front page"
+    "page-not-found": "Ukurasa haujapatikana",
+    "go-to-front-page": "Nenda kwenye ukurasa wa mbele"
   },
   "social-row": {
     "cta": {
-      "tweet-a-thanks": "If you read this far, tweet to the author to show them you care. {{ <0> }}Tweet a thanks{{ </0> }}",
-      "tweet-it": "If this article was helpful, {{ <0> }}tweet it{{ </0> }}."
+      "tweet-a-thanks": "Ukisoma hadi hapa, tweet kwa mwandishi ili kumwonyesha kuwa unajali. {{ <0> }}Tweet shukrani{{ </0> }}",
+      "tweet-it": "Ikiwa makala hii ilikuwa na manufaa, {{ <0> }}ishiriki kwa tweet{{ </0> }}."
     },
     "tweets": {
-      "default": "Thank you {{ author }} for writing this helpful article.",
-      "translation": "Thank you {{ author }} for writing this helpful article, and {{ translator }} for translating it."
+      "default": "Asante {{ author }} kwa kuandika makala hii muhimu.",
+      "translation": "Asante {{ author }} kwa kuandika makala hii muhimu, na {{ translator }} kwa kutafsiri."
     }
   },
-  "learn-to-code-cta": "Learn to code for free. freeCodeCamp's open source curriculum has helped more than 40,000 people get jobs as developers. {{ <0> }}Get started{{ </0> }}",
+  "learn-to-code-cta": "Jifunze kuweka msimbo bila malipo. Mtaala wa chanzo wazi wa freeCodeCamp umesaidia zaidi ya watu 40,000 kupata kazi kama wasanidi. {{ <0> }}Anza{{ </0> }}",
   "original-author-translator": {
     "roles": {
-      "author": "Author: {{ name }} ({{ locale }})",
-      "translator": "Translator: {{ name }}"
+      "author": "Mwandishi: {{ name }} ({{ locale }})",
+      "translator": "Mfasiri: {{ name }}"
     },
     "details": {
-      "original-article": "{{ <0> }}Original article:{{ </0> }} {{ title }} by {{ author }}",
-      "translated-by": "{{ <0> }}Translated by:{{ </0> }} {{ translator }}"
+      "original-article": "{{ <0> }}Makala asilia:{{ </0> }} {{ title }} na {{ author }}",
+      "translated-by": "{{ <0> }}Imetafsiriwa na:{{ </0> }} {{ translator }}"
     },
     "locales": {
-      "arabic": "Arabic",
-      "bengali": "Bengali",
-      "chinese": "Chinese",
-      "english": "English",
-      "espanol": "Spanish",
-      "italian": "Italian",
-      "japanese": "Japanese",
-      "portuguese": "Portuguese",
-      "swahili": "Swahili",
-      "urdu": "Urdu"
+      "arabic": "Kiarabu",
+      "bengali": "Kibengali",
+      "chinese": "Kichina",
+      "english": "Kiingereza",
+      "espanol": "Kihispania",
+      "italian": "Kiitaliano",
+      "japanese": "Kijapani",
+      "portuguese": "Kireno",
+      "swahili": "Kiswahili",
+      "urdu": "Kiurdu"
     }
   },
   "embed-title": "Embedded content",
```

Comment:
This string is less important since it's just the title for embedded iframes. But it would be good to translate, too.

Reply:
Done